MiCCDataAnalytics/PowerBI/Voice/Readme.md

```diff
@@ -9,18 +9,22 @@ This repository contains Power BI templates for devices types connected to Voice
 
 The storage mode for this report dashboard is Import
 
+Required MiCC-B 10.0.0.0
```

Comment:
@shrubs-ok  Can we move this statement as the first thing under the title?
I find it a bit confusing when I read it to know which version requirement goes with which thing.
Also, the wording "Required" seems weird. It feels like past tense.  Should we say Requires MiCC-B 10.0.0.0  ?